code/game/machinery/computer/pod.dm

```diff
@@ -1,21 +1,38 @@
 /obj/machinery/computer/pod
 	name = "mass driver launch control"
 	desc = "A combined blastdoor and mass driver control unit."
+	/// Connected mass driver
 	var/obj/machinery/mass_driver/connected = null
-	var/title = "Mass Driver Controls"
+	/// ID of the launch control
 	var/id = 1
-	var/timing = 0
+	/// If the launch timer counts down
+	var/timing = FALSE
+	/// Time before auto launch
 	var/time = 30
+	/// Range in which we search for a mass drivers and poddoors nearby
 	var/range = 4
 
-
 /obj/machinery/computer/pod/Initialize()
 	. = ..()
 	for(var/obj/machinery/mass_driver/M in range(range, src))
 		if(M.id == id)
 			connected = M
+			break
 
-
+/obj/machinery/computer/pod/process(delta_time)
```

Comment:
You should not be processing 24/7. Consider only starting processing when the timer is running and stopping processing when the timer is paused.

Reply:
Resolved.